tests/integration/targets/junos_ospf_interfaces/tests/netconf/parsed.yaml

```diff
@@ -13,7 +13,6 @@
               metric: 5
               priority: 3
```

Comment:
@rohitthakur2590 can we keep a test for this option to make sure that it works until it is removed?